alacritty/src/cli.rs

```diff
@@ -294,6 +301,25 @@ pub struct WindowOptions {
     pub window_identity: WindowIdentity,
 }
 
+/// Parameters to the `config` IPC subcommand.
+#[cfg(unix)]
+#[derive(Args, Serialize, Deserialize, Default, Debug, Clone, PartialEq, Eq)]
+pub struct IpcConfig {
+    /// Configuration file options [example: cursor.style=Beam].
+    #[clap(required = true, value_name = "CONFIG_OPTIONS")]
+    pub options: Vec<String>,
+
+    /// Window ID for the new config.
+    ///
+    /// Use `-1` to apply this change to all windows.
+    #[clap(short, long, allow_hyphen_values = true, env = "ALACRITTY_WINDOW_ID")]
+    pub window_id: Option<i64>,
+
+    /// Clear all runtime configuration changes.
+    #[clap(short, long, conflicts_with = "options")]
+    pub reset: bool,
```

Comment:
I was thinking more of having a dedicated binding rather than IPC, but I don't really mind either way.

Reply:
Resetting through IPC seems to make sense when it is set through IPC. This way people can easily have both.